content/sensu-go/6.3/observability-pipeline/observe-schedule/business-service-management.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Manage Business Services"
+description: "Sensu's business service management allows you to configure service components and rule templates for monitoring your business services."
+product: "Sensu Go"
+version: "6.3"
+weight: 55
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.3:
+    parent: observe-schedule
+---
+
+Sensu's business service management allows you to monitor every component in your system with a top-down approach that produces meaningful alerts, prevents alert fatigue, and helps you focus on your core business services.
+
+An example of a business service might be a company website.
+The website itself might have three components: the primary webserver that publishes website pages, a backup webserver in case the primary webserver fails, and an inventory database for the shop section of the website.
+At least one webserver and the database must be in an OK state for the website to be fully available.
+
+Business service management allows you to create customized monitoring rules that specify the threshold for taking action for different service components as well as what action to take for different service components.
+
+To continue the company website example, failure of the primary webserver does not affect the website availability because the backup webserver automatically takes over.
+In this scenario, a monitoring rule might create a ticket to address the next workday.
+On the other hand, failure of both webservers or the inventory database does affect website availability, so another monitoring rule that triggers an alert to the on-call operator might apply in that scenario.
+
+Business service management requires two resources that work together to achieve top-down monitoring: [components][1] and [rule templates][2].
+Components are the elements that make up your business services.
+Rule templates define the monitoring rules that produce events for components based on customized evaluation expressions.
+
+## Example component and rule template definitions
+
+Although you can configure components and rule templates within the web UI, both are Sensu resources with complete definitions.
+
+Here is an example component definition that references the `status-threshold` rule template:
+
```

Comment:
These examples should be replaced with examples to fit the company website scenario described in the introduction.